### PR TITLE
Convert MemberList to TS

### PR DESCRIPTION
**Pull request by SimonBrandner**

### CODE REVIEW

**Comment by t3chguy** [comment]
CI be v unhappy

File: src/components/views/rooms/MemberList.tsx [hunk 1]
```diff
@@ -40,41 +52,62 @@ const SHOW_MORE_INCREMENT = 100;
...
 
         cli.on("Room", this.onRoom); // invites & joining after peek
         const enablePresenceByHsUrl = SdkConfig.get()["enable_presence_by_hs_url"];
         const hsUrl = MatrixClientPeg.get().baseUrl;
-        this._showPresence = true;
+        this.showPresence = true;
         if (enablePresenceByHsUrl && enablePresenceByHsUrl[hsUrl] !== undefined) {
-            this._showPresence = enablePresenceByHsUrl[hsUrl];
+            this.showPresence = enablePresenceByHsUrl[hsUrl];
         }
```
Comment (t3chguy) [inline src/components/views/rooms/MemberList.tsx:95]: Think this would be cleaner as 

```
this.showPresence = enablePresenceByHsUrl?.[hsUrl] ?? true;
```

**Review by t3chguy — APPROVED** [review]
LGTM otherwise

File: src/components/views/rooms/MemberList.tsx [hunk 2]
```diff
@@ -91,7 +91,7 @@ export default class MemberList extends React.Component<IProps, IState> {
         const hsUrl = MatrixClientPeg.get().baseUrl;
         this.showPresence = true;
         if (enablePresenceByHsUrl && enablePresenceByHsUrl[hsUrl] !== undefined) {
-            this.showPresence = enablePresenceByHsUrl[hsUrl];
+            this.showPresence = enablePresenceByHsUrl?.[hsUrl] ?? true;
```
Comment (t3chguy) [inline src/components/views/rooms/MemberList.tsx:94]: the suggestion replaced the 4 lines, not just the one. The if is no longer needed

Comment (SimonBrandner) [reply src/components/views/rooms/MemberList.tsx]: Sorry :man_facepalming:

**Pull request merged**